Tribler/community/multichain/community.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, *args, **kwargs):
         self._private_key = self.my_member.private_key
         self._public_key = self.my_member.public_key
         self.persistence = MultiChainDB(self.dispersy, self.dispersy.working_directory)
+        self.logger.debug("The multichain community started with Public Key: " + base64.encodestring(self._public_key))
```

Comment:
You may want to use `self.logger.debug("The multichain community started with Public Key: %s", base64.encodestring(self._public_key))` as this is considered "lazy logging"